data/help.txt

```diff
@@ -83,10 +83,11 @@ help "outfitter"
 help "outfitter 2"
 	`Sometimes you may want to buy some outfits directly into your cargo hold instead of having them installed. Perhaps a mission has asked you to retrieve some outfits or you want to carry extra ammunition to refill your ships somewhere that does not stock those weapons. To accomplish this, you need to switch out of installation mode into 'purchase to cargo' mode. This can be achieved by either checking the "show outfits in cargo" box at the bottom right, or control-clicking on your selected ship to deselect it.`
 	`When you are in 'purchase to cargo' mode, your ship information on the right hand side will be replaced by information on how much cargo space you currently have available.`
+	`Outfits that are in your cargo hold can be (re)installed at any outfitter without cost.`
 
 help "outfitter 3"
-	`You can also uninstall outfits directly into your cargo hold by selecting the outfit and pressing U. This can be useful for if you want to shuffle outfits between ships or simply try on different outfits. Outfits that are in your cargo hold can be installed at any outfitter without cost.`
-	`If you sell an outfit at an outfitter, then you can immediately buy it back for the same price that you sold it, so long as you do not leave the planet. This means that you do not necessarily need to worry about uninstalling outfits into cargo when you are changing outfits around.`
+	`You can also uninstall outfits directly into planetary storage by selecting the outfit and pressing U. This can be useful for if you want to shuffle outfits between ships, store outfits for future use or simply try on different outfits. Outfits that are in planetary storage can (re)installed at any outfitter without cost. Outfits in storage can also be transferred to cargo when you go to 'purchase to cargo' mode.`
```

Comment:
```suggestion
	`You can also uninstall outfits directly into planetary storage by selecting the outfit and pressing U. This can be useful for if you want to shuffle outfits between ships, store outfits for future use, or simply try on different outfits. Outfits in storage can also be transferred to cargo when you go to 'purchase to cargo' mode. Outfits that are in planetary storage or cargo can be (re)installed at any outfitter without cost.`
```